python/grapher.go

```diff
@@ -13,6 +14,7 @@ import (
 
 	"sourcegraph.com/sourcegraph/srclib/graph"
 	"sourcegraph.com/sourcegraph/srclib/unit"
+	"sourcegraph.com/sourcegraph/toolchain"
```

Comment:
I'll sort this out. It's actually `goimports` that's adding the wrong path. No idea why it adds the wrong one yet.